make_md.py

```diff
@@ -49,7 +54,14 @@ def convert_from_bib(myline):
         print(myentry)
         print("We are in trouble ! ")
     if "eprint" in myentry_dict:
-        return "["+myentry_dict["title"]+"](https://arxiv.org/abs/"+myentry_dict["eprint"]+")"
+        paper=""
+        if "doi" in myentry_dict:
+            paper=" [Published](https://doi.org/{0})".format( myentry_dict["doi"] )
```

Comment:
Without any sort of deliminator this is very hard to visually distinguish.
```suggestion
            paper=f" [[Published](https://doi.org/{myentry_dict['doi']})]"
```

Reply:
I leaves those out. this can be done on the master with changing this everywhere applicable

Reply:
I mean the f-string, sorry for the confusion

Reply:
Sure, this can be done everywhere in a single sweep, but also doesn't hurt to write modern Python now. But to answer your question below

> is the f-string change mandatory to move forward at this point?

No, it isn't required.